Source/src/core/GameObject.cpp

```diff
@@ -810,6 +810,23 @@ void Hachiko::GameObject::ChangeTintColor(float4 color, bool include_children)
     }
 }
 
+void Hachiko::GameObject::ChangeDissolveProgress(float progress, bool include_children) 
+{
+    std::vector<ComponentMeshRenderer*> v_mesh_renderer = GetComponents<ComponentMeshRenderer>();
+    for (int i = 0; i < v_mesh_renderer.size(); ++i)
+    {
+        v_mesh_renderer[i]->SetDissolveProgress(progress);
+    }
```

Comment:
In cases where the index is not use, it's better to opt in for range based for loop just like you did for children on the following lines.

Reply:
Oka, but why is better?